rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -113,6 +113,8 @@ components:
       headers:
         'Content-Version':
           $ref: '#/components/headers/ContentVersion'
+        'Location':
+          $ref: '#/components/headers/Location'
```

Comment:
❓ Shouldn't we have this in the body?

Reply:
I think the standard is to have a `Location` header, but I'm not confident. https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/client/RestTemplate.html#postForLocation-java.lang.String-java.lang.Object-java.util.Map-

Reply:
Actually. Looks like I'm wrong. It should be in the response body. 
https://stackoverflow.com/questions/26199228/is-the-use-of-location-header-in-http-202-response-rfc-compliant
>the Location header should not be used in 202 Accepted response

Reply:
I'm getting conflicting answers:
>The HTTP spec says that a 202 Accepted response SHOULD include an indication of the request’s current status.... It may also be beneficial to clients to include a Location header with a URI that can be used later to check on the status of the request.
https://benramsey.com/blog/2008/04/http-status-201-created-vs-202-accepted/

Reply:
Based on the official http 1.1 RFC (see https://tools.ietf.org/html/rfc7231#section-7.1.2), the `Location` header should only be used with status code 201 and the 3xx series. So in this case I would recommend returning the URI in the response's body.
However, if nobody needs it right now, we could also postpone the decision and wait until we figure out what information we really want to return to the client when an ingest request is submitted.

Reply:
I think the question is why are we returning 202 and not 201? 202 means it has been accepted and is being worked on. Normally that would mean that the URL is not yet available to retrieve anything. However in this case, although not completely created, the resource that was passed is available at that URL and some of the derived info will eventually be available as they get added. I would actually say it is created.

Reply:
I don't this that it "is available" because It will not be available to anybody other than Transformation Service. If you tried to use the retrieve URL immediately, it would return Not Authorized.

Reply:
I don't think it should. It might be that until we know more the authorization for it is super high and therefore you get not authorized but the fact is that it is there. In any case, if it ain't there then "Not Authorized" would be wrong as you should return 404

Reply:
We did have a short discussion on this yesterday and I forgot to update this thread. We decided to hold off on changing anything at this point since there's only one consumer of this API and it only checks the status code series (2xx, 4xx or 5xx) right now. When we get a better understanding of what's needed and needs to be returned, we will update the API accordingly.